src/calendar/Calendar.ts

```diff
@@ -359,9 +374,12 @@ export default class Calendar<P extends CalendarProperties = CalendarProperties>
 	}
 
 	protected render(): DNode {
+		const calendarMessages = this.localizeBundle(calendarBundle);
+		const commonMessages = this.localizeBundle(commonBundle);
```

Comment:
It would be ideal if these were only loaded when their corresponding properties are unavailable. As it stands now, locale-specific messages will always be loaded, possibly via another HTTP request, regardless of whether they are actually needed.